src/devsite-paths.json

```
@@ -54,5 +54,12 @@
         "owner": "AdobeDocs",
         "root": "/src/pages/",
         "branch": "eds-migration"
+    },
+    {
+        "pathPrefix": "/Developer-Distribution-Experience-Cloud/docs",
```

```suggestion
        "pathPrefix": "/developer-distribution/experience-cloud/docs/",
```

from: https://github.com/AdobeDocs/Developer-Distribution-Experience-Cloud/blob/main/gatsby-config.js#L15